diff --git a/src/xmlrpc_string.c b/src/xmlrpc_string.c
index 2da61e4..1797790 100644
--- a/src/xmlrpc_string.c
+++ b/src/xmlrpc_string.c
@@ -746,15 +746,15 @@ stringNew(xmlrpc_env *     const envP,
           enum crTreatment const crTreatment,
           xmlrpc_value **  const valPP) {
 
-    fprintf(stderr, "stringNew entered\n");
+//    fprintf(stderr, "stringNew entered\n");
     xmlrpc_value * valP;
 
     xmlrpc_validate_utf8(envP, value, length);
 
     if (!envP->fault_occurred) {
-        fprintf(stderr, "Going to xmlrpc_createXmlrpcValue\n");
+//        fprintf(stderr, "Going to xmlrpc_createXmlrpcValue\n");
         xmlrpc_createXmlrpcValue(envP, &valP);
-        fprintf(stderr, "Back from createXmlrpcValue\n");
+//        fprintf(stderr, "Back from createXmlrpcValue\n");
 
         if (!envP->fault_occurred) {
             valP->_type = XMLRPC_TYPE_STRING;
@@ -774,7 +774,7 @@ stringNew(xmlrpc_env *     const envP,
                 *valPP = valP;
         }
     }
-    fprintf(stderr, "stringNew exiting\n");
+//    fprintf(stderr, "stringNew exiting\n");
 }
 
 
